templates/documentation/cmip6_downscaled.html

```diff
@@ -0,0 +1,149 @@
+{% extends 'base.html' %} {% block content %}
+<h2>CMIP6, Downscaled</h2>
+
+<p>
+  This endpoint provides access to daily CMIP6 data that have been bias-adjusted and downscaled to 4km using ERA5
+  reference data. Select models and scenarios are available. Read more about our CMIP6 subset
```

Comment:
Slight preference for stating the number of models and scenarios here, along with a phrase that conveys we don't have the superset product of all combinations: availability varies by model, etc.

Reply:
Fixed in 9de32ba, I think!